backend-go/internal/core/hello/handler.go

```diff
@@ -0,0 +1,38 @@
+package hello
+
+import (
+	"github.com/labstack/echo/v4"
+	"github.com/yorkie-team/codepair/backend/api/codepair/v1/models"
+	"github.com/yorkie-team/codepair/backend/internal/transport/rest"
+)
+
+type Handler struct {
+	helloService Service
```

Comment:
```suggestion
	helloService *Service
```

How about having fields with pointer types instead of value types?